ui/src/components/header/SamlDomainSwitcher.vue

```diff
@@ -22,6 +22,11 @@
       :loading="loading"
       :defaultValue="currentAccount"
       :value="currentAccount"
+      showSearch
+      optionFilterProp="children"
+      :filterOption="(input, option) => {
+        return option.componentOptions.children[0].text.toLowerCase().indexOf(input.toLowerCase()) >= 0
+      }"
```

Comment:
@davidjumani this code is common across all select items. Is it possible to extract this and have a custom select component?

Reply:
I think that might involve replicating every prop used which could potentially break things. @utchoang ?